src/tiled/mainwindow.cpp

```diff
@@ -1187,14 +1187,19 @@ void MainWindow::addExternalTileset()
     if (!mMapDocument)
         return;
 
-    const QString start = fileDialogStartLocation();
+    Preferences *prefs = Preferences::instance();
+
+    const QString start = prefs->lastPath(Preferences::ExternalTileset);
```

Comment:
Please still use `fileDialogStartLocation()` if the last path is empty (the first time people use it).